docs/enterprise-appstore/enterprise-app-store-profile.md

```diff
@@ -0,0 +1,272 @@
+
+#### 3. **Delete**
+The "Delete" action permanently removes a profile from the Enterprise App Store. Once a profile is deleted, all associated data, including uploaded binaries and settings, are also erased.
+
+:::info
+
+Unlike the profiles from other modules within the Appcircle, Enterprise App Store profiles cannot be named or renamed manually. The profiles will display the binary names instead.
+
+:::
+
+## Profile Settings
+
+After you have created your profile, it is time to configure it and send it to different users and channels.
+
+You can reach your profile information and add users for them to gain access to the Live and Beta channels by clicking the **Settings** button.
+
+<Screenshot url='https://cdn.appcircle.io/docs/assets/BE-4225-profile4.png' />
+
+### Information
+
+You can copy your Profile ID from the Info tab by clicking the copy icon on the right side of your displayed ID.
+
+<Screenshot url="https://cdn.appcircle.io/docs/assets/BE-4225-profilesettings.png" />
+
+### Manage Access
+
+You can add multiple users to the Beta and Live channels by hitting the Tab key.
+
+<Screenshot url="https://cdn.appcircle.io/docs/assets/BE-4225-notify1.png" />
+
+:::important
+
+To configure the Manage Access tab, the authentication method for your Enterprise Store must be either SSO or LDAP. For more information about authentication types, please refer to the [Store Settings](/enterprise-appstore/store-settings#store-authentication) documentation.
+
+:::
+
+
+:::caution
+If you work for a sub-organization, you will only have visibility of the apps within that organization. However, if an app is published, it will also be displayed in the parent organization's showcase on the Enterprise App Store.
+
+To manage user permissions within the Enterprise App Store, **Okta SAML** can be utilized. By configuring authorization groups through **Okta** and subsequently using these group names on **Alpha** or **Beta** channels via Appcircle, only relevant users will have access to them.
+
+For further details, please refer to the document: [Okta Managing User Groups](https://docs.appcircle.io/account/my-organization/sso-providers-configuration/sso-login/okta-saml#okta-managing-user-groups).
+:::
+
+:::info
+If Beta Channel Access is not set, the versions marked as the Beta channel will not be visible to anyone by default.
+
+If the Live Channel Access is not set, the versions marked as the Live channel will be visible to everyone by default.
+
+:::
+
+You can add defined user groups from your provider after configuration.
+
+<Screenshot url="https://cdn.appcircle.io/docs/assets/2812-entstore-okta-group-new.png" />
+
+:::tip 
+
+You can also update the logo of your Enterprise App Store profile by clicking on the logo icon. Please note that this will not affect your Enterprise Store login customization.
```

Comment:
```suggestion
The logo of the Enterprise App Store profile can also be updated by clicking on the logo icon. Please note that this will not affect the customization of your Enterprise Store login.
```